src/CollapsibleCard/index.js

```diff
@@ -3,17 +3,21 @@ import PropTypes from "prop-types";
 import cc from "classcat";
 import Row from "../Row";
 import IconButton from "../IconButton";
+import DisabledShim from "../DisabledShim";
+
+const noop = () => {};
```

Comment:
noop is a named function - sort of a micro optimization for JS garbage collection (`() => {}` functions are anonymous and redefined each pass)